packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -332,6 +332,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     markNeedsTextLayout();
   }
 
+  /// Character used to obscure text if [obscureText] is true.
```

Comment:
Maybe also mention this can't be null, in case someone uses their own widget to create a `RenderEditable` (instead of using an `EditableText`)?